bindings/python/iota_sdk/utils.py

```diff
@@ -1,14 +1,17 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
+from __future__ import annotations
 from iota_sdk import call_utils_method
-from iota_sdk.types.address import Ed25519Address
 from iota_sdk.types.signature import Ed25519Signature
 from iota_sdk.types.common import HexStr
 from iota_sdk.types.output_id import OutputId
 from json import dumps, loads
-from typing import Any, Dict
+from typing import Any, Dict, TYPE_CHECKING
 
+# Required to prevent circular import
+if TYPE_CHECKING:
```

Comment:
weird

Reply:
Yes, but there doesn't seem to be a better alternative https://mypy.readthedocs.io/en/stable/runtime_troubles.html#typing-type-checking